app/Phragile/TransactionFilter.php

```diff
@@ -1,6 +1,7 @@
 <?php
 namespace Phragile;
 
+// TODO: rename to e.g. TransactionDataFilter
```

Comment:
hmm, but shouldn't it actually be `TransactionRawDataFilter` as it deals with Phabricator's data (per `TransactionRawDataProcessor`)?